src/Translation_unit.ml

```diff
@@ -115,6 +115,19 @@ let lf_to_crlf =
     (String.Escaping.escape_gen_exn ~escapeworthy_map:[('\n', '\n')]
        ~escape_char:'\r')
 
+let crlf_translated_to_lf () =
+  let filename = "ocamlformat_crlf_testing" in
+  let binary = true and append = false in
+  let write oc = Stdio.Out_channel.output_string oc "\r\n" in
+  Stdio.Out_channel.with_file ~binary ~append filename ~f:write ;
+  let read = Stdio.In_channel.input_line ~fix_win_eol:false in
```

Comment:
Since we are more concerned with the output, I wonder if it would make sense to test by writing in text mode and reading in binary.